### PR TITLE
[ign-math4] Update BitBucket links

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

File: include/ignition/math/Material.hh [hunk 1]
```diff
@@ -41,7 +41,7 @@ namespace ignition
     /// enum.
     ///
     /// This class will replace the
-    /// [MaterialDensity class](https://bitbucket.org/ignitionrobotics/ign-common/src/ign-common1/include/ignition/common/MaterialDensity.hh)
+    /// [MaterialDensity class](https://github.com/ignitionrobotics/ign-common/blob/ign-common1/include/ignition/common/MaterialDensity.hh)
```
Comment (ahcorde) [inline include/ignition/math/Material.hh:44]:
```suggestion
    /// [MaterialDensity class](https://github.com/ignitionrobotics/ign-common/blob/ign-common1/include/ignition/common/MaterialDensity.hh)
```

File: README.md [hunk 2]
```diff
@@ -11,14 +11,6 @@ classes and functions used in other Ignition libraries and programs.
 
 ## Continuous integration
 
-This project uses [appveyor](https://ci.appveyor.com/project/scpeters/ign-math/history)
```
Comment (ahcorde) [inline README.md:14]: why did you remove this comment? appveyor is still configured for windows, right?

Comment (chapulina) [reply README.md:14]: I've been removing Appveyor and Drone.io because I assumed we'd move to GitHub pipelines. But we should probably get @scpeters 's ok on this. In any case, the current links are "invalid" until we re-setup Appveyor / Drone for GitHub.
